src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+@use 'variables' as v;
+
+.calendar {
+  margin: 0 auto;
+  width: 706px;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @each $day, $i in v.$week {
+    &--start-day-#{$day} :first-child {
```

Comment:
The selector `&--start-day-#{$day} :first-child` has a space before `:first-child`, which increases specificity unnecessarily. Consider removing the space to ensure the pseudo-class applies directly to the element with the `&--start-day-#{$day}` class.